astropy/units/equivalencies.py

```diff
@@ -503,9 +503,31 @@ def from_vel_en(x):
         return resten * ((1-voverc) / (1+(voverc)))**0.5
 
     return Equivalency([(si.Hz, si.km/si.s, to_vel_freq, from_vel_freq),
-            (si.AA, si.km/si.s, to_vel_wav, from_vel_wav),
-            (si.eV, si.km/si.s, to_vel_en, from_vel_en),
-            ], "doppler_relativistic", {'rest': rest})
+                        (si.AA, si.km/si.s, to_vel_wav, from_vel_wav),
+                        (si.eV, si.km/si.s, to_vel_en, from_vel_en),
+                        ], "doppler_relativistic", {'rest': rest})
+
+
+def doppler_redshift():
+    """
+    Returns the equivalence between Doppler redshift (unitless
+    or `astropy.cosmology.units.redshift`) and radial velocity.
```

Comment:
`astropy.cosmology.units.redshift` is not actually a Doppler redshift, but a cosmological redshift (https://en.wikipedia.org/wiki/Redshift). However, when interpreting a cosmological redshift as a recessional velocity, the math is the same. I would just change the phrasing to something like

```suggestion
    Returns the equivalence between Doppler redshift and radial velocity.
    For quantities with units of `astropy.cosmology.units.redshift`, this is also
    the equivalence between cosmological redshift and recessional velocity.
```